src/Application/test/CalculatorClassGTest.cpp

```diff
@@ -25,61 +30,166 @@ using namespace testing;
 class whenTestingCalculator:public ::testing::Test
 {
     public:
-        Calculator* calculator;
+        CalculatorApplicationFactory* calculator;
```

Comment:
rename to calculatorFactory

Reply:
Actually, it doesn't look like you are using this at all anymore. You should remove it and then create a new test fixture to unit test for the calculator factory.